priv/rebar3_bsp_launcher.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -eu
+
+# Rebind stdio fds to new ones
+exec 3<&0 # stdin dup'd to fd 3
+exec 4<&1 # stdout dup'd to fd 4
+exec 5<&2 # stderr dup'd to fd 5
+
+# Redirect stdio fds to safety
+exec 0</dev/null # no input on fd 0
+exec 1>>/tmp/rebar3_bsp.log # log output
+exec 2>&1 # redirect stderr output to stdout
+
+# Inform the BSP plugin of our shenanigans
+export REBAR3_BSP_IO_FDS="3 4 5"
```

Comment:
It looks like we ignore the error fd (`open_port` only accepts `in` and `out`), so we should probably not pass it.